widgets/opal/switchbutton/org.eclipse.nebula.widgets.opal.switchbutton/src/org/eclipse/nebula/widgets/opal/switchbutton/SwitchButton.java

```diff
@@ -161,38 +155,33 @@
 	public SwitchButton(final Composite parent, final int style) {
 		super(parent, style | SWT.DOUBLE_BUFFERED);
 
-		selection = false;
+		selection = true;

```

Comment:
I thing "false" as a default is more like it is with standard swt widget, e.g. a checkbox is initially unselected, a toogle button is not toggled and so on...

Reply:
Good suggestion, it is solved in the next commit